database/oracle/oracle.Dockerfile

```
@@ -0,0 +1,74 @@
+# Use Ubuntu 24.04 as the base image
+FROM ubuntu:24.04
+
+# Set environment variables
+ENV ORACLE_BASE=/opt/oracle \
+    ORACLE_HOME=/opt/oracle/product/19c/dbhome_1 \
+    ORACLE_SID=ORCLCDB \
+    PATH=$PATH:/opt/oracle/product/19c/dbhome_1/bin
+
+# Update Ubuntu and install necessary packages
+RUN apt-get update && \
```

Hadolint

[warning] Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`

(DL3008)